project/SparkBuild.scala

```diff
@@ -390,7 +390,7 @@ object SparkBuild extends PomBuild {
   val mimaProjects = allProjects.filterNot { x =>
     Seq(
       spark, hive, hiveThriftServer, repl, networkCommon, networkShuffle, networkYarn,
-      unsafe, tags, tokenProviderKafka010, sqlKafka010, connect
+      unsafe, tags, tokenProviderKafka010, sqlKafka010, connect, protobuf
```

Comment:
What is this for? Avro is not included here.

Reply:
@rangadi it is part of shading google-protobuf jar through sbt. so passing protobuf module through MimaBuild(is a tool for identifying [binary incompatibilities](https://docs.scala-lang.org/overviews/core/binary-compatibility-for-library-authors.html) in Scala libraries.).

Reply:
Thanks @SandishKumarHN. How do we trigger this? With sbt build?

Reply:
@rangadi there is file to trigger, 
```
dev/mima
```